plugins/homebrew/personal_access_token.go

```diff
@@ -0,0 +1,41 @@
+package homebrew
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func PersonalAccessToken() schema.CredentialType {
```

Comment:
I was wondering if we should re-use the definition of the GitHub Personal Access Token from the GitHub plugin, but considering GitHub CLI supports enterprise hosting while Brew only uses https://github.com I think creating a separate definition here like you did is the best approach. :+1: